site/content/docs/mdx.md

```diff
@@ -0,0 +1,288 @@
+import { Callout } from '../components/TempCallout.jsx'
+
+# MDX
+
+Flowershow parses all of your Markdown files as MDX. This means you not only can write your content using good old Markdown, but also extend it with JSX, JavaScript expressions, and `import` and `export` statements. It's basically Markdown on steroids 😀💪!
```

Comment:
Context: This page is excellent. Comments here are more suggestions unless flagged as otherwise with 🚩

Personally, I might add a tiny bit about why this is powerful. e.g. "What this means is you can enrich your content with the full power of javascript and modern web technologies. [maybe add an examples e.g. adding rich dynamic visualizations or embedding data tables ...]